### PR TITLE
PMM-6461 pg_stat_monitor

### PR DESCRIPTION
**Pull request by JiriCtvrtka**
[PMM-6461](https://jira.percona.com/browse/PMM-6461)

- [x] Build: https://github.com/Percona-Lab/pmm-submodules/pull/1132

- [x] Links to other linked pull requests (optional).
PMM-Admin: https://github.com/percona/pmm-admin/pull/118
---
- [x] Tests passed.
- [ ] Feature build pass.
- [x] (Re)requested review.
- [ ] Fix conflicts with target branch.
- [ ] Update API dependency.
- [ ] Update jira ticket description if needed.
- [ ] Attach screenshots/console output to confirm new behavior to jira ticket, if applicable.

### CODE REVIEW

**Review by BupycHuk — CHANGES_REQUESTED** [review]
Your PR contains a lot of changes from other PRs, please re-create a PR with only your changes from PMM-2.0.

**Comment by JiriCtvrtka** [comment]
@BupycHuk Yes, true. My fault during creating branch. Fixed. Thanks.

File: models/database.go [hunk 1]
```diff
@@ -485,6 +485,15 @@ func setupFixture1(q *reform.Querier, username, password string) error {
 	if err != nil {
 		return err
 	}
+	_, err = CreateAgent(q, QANPostgreSQLPgStatMonitorAgentType, &CreateAgentParams{
```
Comment (BupycHuk) [inline models/database.go:488]: let's remove it

**Review by BupycHuk — CHANGES_REQUESTED** [review]
let's remove pg_stat_monitor agent from fixtures and looks like all other changes are based on that.

**Pull request closed without merging**